index.html

```diff
@@ -202,6 +202,58 @@ <h3 class="title">Multy-Post Stories</h3>
       </div>
     </div>
   </section>
+
+  <section class="contact">
+    <h1>CONTACT US</h1>
+    <p>If you have an application you are interested in developing, a feature that you
+       need built or a project that needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/xbjwvedl" method="post">
+            <ul>
+              <li><input type="text" placeholder="First Name" maxlength="30" required style="width: 260px;height: 30px;">
+              </li>
+             <li><input type="email" placeholder="E-mail" required style="width: 260px;height: 30px;"></li>
+             <li><textarea name="msg" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required style="width: 260px;"></textarea></li>  
+            </ul>
+             <button class="s-button" type="summit" style="width: 100px;height: 43px;margin-left: 70px;">See Project</button>
+        </form>
+  </section>
```

Comment:
Hello Duncan 😃 here in this piece of code I recommend you to check on the identation between the lines. the spacing should be of 2 spaces between parents and children tags, it would be good if you can asign the styles with clases and not directly inside the tag.

For the identation in the tags it should look like this:
<p></p>
<form>
  <ul>
    <li></li>
  </ul>
</form>

Check in the rest of the code for this issues